modulecheck-parsing/java/src/test/kotlin/modulecheck/parsing/java/JavaFileTest.kt

```diff
@@ -359,7 +364,7 @@ internal class JavaFileTest : ProjectTest() {
 
         public class ParsedClass {
 
-          public com.lib1.Lib1Class foo() { return Lib1Class(); }
+          public com.lib1.Lib1Class foo() { return com.lib1.Lib1Class(); }
```

Comment:
This wasn't valid either, as the previous return of `Lib1Class()` wouldn't have resolved.